### PR TITLE
Fixed segfault in stats where container had netNS none or from container

### PR DESCRIPTION
**Pull request by haircommander**
Signed-off-by: haircommander <pehunt@redhat.com>

### CODE REVIEW

File: libpod/networking_linux.go [hunk 1]
```diff
@@ -249,9 +249,33 @@ func (r *Runtime) teardownNetNS(ctr *Container) error {
 	return nil
 }
 
+func getContainerNetNS(ctr *Container) (string, error) {
+	if ctr.state.NetNS != nil {
+		return ctr.state.NetNS.Path(), nil
+	}
+	if ctr.config.NetNsCtr != "" {
+		c, err := ctr.runtime.LookupContainer(ctr.config.NetNsCtr)
```
Comment (mheon) [inline libpod/networking_linux.go]: NetNSCtr is required to be the full ID, to set can use GetContainer() here, which should be a bit faster faster

File: libpod/networking_linux.go [hunk 2]
```diff
@@ -249,9 +249,33 @@ func (r *Runtime) teardownNetNS(ctr *Container) error {
...
+		}
+		if err = c.syncContainer(); err != nil {
+			return "", err
+		}
+		return c.state.NetNS.Path(), nil
+	}
+	return "", nil
+}
+
 func getContainerNetIO(ctr *Container) (*netlink.LinkStatistics, error) {
```
Comment (mheon) [inline libpod/networking_linux.go]: Maybe a comment here that nil, nil is a valid return if the container has no netns?

Comment (haircommander) [reply libpod/networking_linux.go]: fixed at return statement

**Comment by mheon** [comment]
LGTM

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**